sig-instrumentation/charter.md

```diff
@@ -0,0 +1,52 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers best practices for cluster observability through metrics, logging, and
+events across all Kubernetes components and development of relevant components.
+
+### In scope
+
+Coordinating metric requirements of different SIGs for other components through finding common APIs.
+
+[Link to SIG section in sigs.yaml][sigs.yaml].
+
+#### Code, Binaries and Services
+
+- Components required for any Kubernetes cluster in regards to observability.
+- Interfaces/API definitions required for any Kubernetes cluster in regards to observability.
+- Well established but optional components for Kubernetes clusters, if endorsed by member.
+- Guidance for instrumentation in order to ensure consistent and high quality instrumentation of core components.
+
+#### Cross-cutting and Externally Facing Processes
+
+n/a
```

Comment:
two concerns here:

- being the "reviewers" for instrumentation (somewhat akin to api-reviewers)
- being involved in things like node monitoring, where we collaborate with another sig on the details of monitoring parts of kubernetes

Reply:
agreed, I think people should come to this SIG for advice on what metrics to export and the best practices on the dimensions